index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/8b2919a184.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <!-- ------------------The Nav Bar-------------------->
+    <header>
+      <nav>
+        <ul>
+          <li><a href="#">My Logo</a></li>
+          <li>
+            <a href="#"><i class="fa-solid fa-bars"></i></a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Good job using the header tag :clap: , however, :thinking: kindly make sure the logo is out of the nav, for semantic purpose, you can do as follow:
`<header>`
`<a href="#">My Logo</a>`
`<nav><i class="fa-solid fa-bars"></nav>`
`</header>`

Reply:
@GedeonTS, I do not understand this issue. Do you mean the nav should be on its own and the logo on its own?